src/state/README.md

```diff
@@ -1,14 +1,36 @@
 # State Manager
+The `StateManager` is a class that manages a group of subscribable [`Property`](./Property.js) instances.
+The provides functionality for registration of new properties, and for subscribing to existing properties.
```

Comment:
This class provides functionality for registration of new properties and subscription to existing properties.